manifests/d/DIG/PWAutoLogin/0.2.3/DIG.PWAutoLogin.locale.en-US.yaml

```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: DIG.PWAutoLogin
+PackageVersion: 0.2.3
+PackageLocale: en-US
+Publisher: DIG
+PublisherUrl: https://github.com/DIG-/pw-auto-login
+PublisherSupportUrl: https://github.com/DIG-/pw-auto-login/issues
+Author: Gustavo de Oliveira Silva (DIG)
+PackageName: PW Auto Login
+PackageUrl: https://github.com/DIG-/pw-auto-login
+License: CC BY-ND 4.0
+LicenseUrl: https://github.com/DIG-/pw-auto-login/blob/0.2.3/license.txt
+Copyright: DIG
+ShortDescription: PW Auto Login Setup

```

Comment:
Please provide a better description of what this application is, what it does, or what it is used for